pkg/operator/config/feature-gates.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 NVIDIA CORPORATION
```

Comment:
Let's use underscore instead of dash (-) in the file name, this seems to be the convention

Reply:
:+1: